mkosi/__init__.py

```diff
@@ -447,34 +463,37 @@ def run_build_scripts(state: MkosiState) -> None:
         )
 
     with (
-        mount_build_overlay(state),\
-        mount_volatile_overlay(state),\
-        finalize_chroot_scripts(state) as cd\
+        mount_build_overlay(state),
+        mount_volatile_overlay(state),
+        finalize_chroot_scripts(state) as cd,
```

Comment:
I was hesitant about this, but it seems to work on the python 3.9 I tested on a Debian bullseye machine, so I think we should try this and see if anybody complains. We should add a warning about this in the news, though.

Reply:
Yeah. I was surprised, but 3.8 fails and 3.9 works, so I think this should be fine.